README.md

```diff
@@ -1,7 +1,19 @@
-# SVGDigitizer — Extract (x,y) Data Points from SVG files
+<p align="center">
+    <img alt="logo" src="./logo.png" width="300px">
+</p>
 
-![Logo](https://raw.githubusercontent.com/echemdb/svgdigitizer/master/doc/files/logo/logo.png)
+<h1><p align="center">svgdigitizer</p></h1>
 
+<p align="center">
+  <img src="https://img.shields.io/badge/License-GPL_3.0_or_later-blue.svg" alt="License: GPL 3.0 or later">
+  <a href="https://doi.org/10.5281/zenodo.8428961"><img src="https://zenodo.org/badge/DOI/10.5281/zenodo.8428961.svg" alt="DOI"></a>
+</p>
+
+<p align="center">Extract (x,y) data points from SVG files</p>
+<hr>
+
+<p align="center"><img src="https://github.com/user-attachments/assets/e8c194f1-6061-43b5-871e-7498e7f76414" width="80%">

```

Comment:
This looks odd in the preview. Is that intentional?

Reply:
Sorry, can you qualify "odd"?

Reply:
Neither in Dark nor Light Mode JSON and CSV are perfectly visible. So I wonder why we need this additional figure in the README or if we should use a simplified version with the JSON and CSV logo, but where the entire figure has a white background.
![image](https://github.com/user-attachments/assets/3b424514-f09c-4116-bf05-f500415eeb7c)

Reply:
Does it look better now?